test/powershell/Modules/Microsoft.PowerShell.Core/Import-Module.Tests.ps1

```diff
@@ -27,6 +39,11 @@
         { $script:module = Import-Module $moduleName -PassThru } | Should Not Throw
```

Comment:
Please add `-ErrorAction Stop` as a failed load is not a terminating error

Reply:
Will add